From: Bruno Haible <bruno@clisp.org>
Date: Wed, 31 Dec 2025 00:37:14 +0100
Subject: [PATCH] build: Avoid link error on native Windows with
 --enable-shared.

Reported by Michele Locati <michele@locati.it> in
<https://lists.gnu.org/archive/html/bug-gettext/2025-12/msg00038.html>.

* gettext-tools/src/country-table.h (country_table, country_table_size): Remove
LIBGETTEXTSRC_DLL_VARIABLE annotation.
---
 gettext-tools/src/country-table.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gettext-tools/src/country-table.h b/gettext-tools/src/country-table.h
index 0f80a6825..6b6280e86 100644
--- a/gettext-tools/src/country-table.h
+++ b/gettext-tools/src/country-table.h
@@ -33,8 +33,8 @@ struct country_table_entry
   const char *english;
 };
 
-extern LIBGETTEXTSRC_DLL_VARIABLE struct country_table_entry country_table[];
-extern LIBGETTEXTSRC_DLL_VARIABLE const size_t country_table_size;
+extern struct country_table_entry country_table[];
+extern const size_t country_table_size;
 
 
 #ifdef __cplusplus
-- 
2.52.0.windows.1

